src/pages/iou/IOUModal.js

```diff
@@ -358,7 +357,7 @@ class IOUModal extends Component {
         return (
             <ScreenWrapper>
                 {({didScreenTransitionEnd}) => (
-                    <KeyboardAvoidingView>
+                    <>
```

Comment:
Doesn't look like this fragment is needed as we are rendering a single element `View`?